client/app/bundles/course/lesson_plan/actions.jsx

```diff
@@ -7,3 +7,101 @@ export function toggleItemTypeVisibility(itemType) {
     itemType,
   };
 }
+
+function combineDateTime(dateSource, timeSource) {
+  const combinedDateTime = dateSource || new Date();
+  if (timeSource) {
+    combinedDateTime.setHours(timeSource.getHours());
+    combinedDateTime.setMinutes(timeSource.getMinutes());
+  }
+  return combinedDateTime;
+}
+
+function patchRequest(endpoint, payload, successHandler, failureHandler) {
+  const csrfToken = document.querySelector('meta[name="csrf-token"]').getAttribute('content');
```

Comment:
We can config axios globally: https://github.com/mzabriskie/axios#config-defaults

Thus I think we might not need this wrapper.